ros_buildfarm/templates/release/rpm/mock_config.cfg.em

```diff
@@ -10,9 +10,7 @@ config_opts['use_bootstrap'] = False
 config_opts['chroot_setup_cmd'] += ' python3-rpm-macros'
 
 # Install weak dependencies to get group members
-config_opts['yum_builddep_opts'] = config_opts.get('yum_builddep_opts', []) + ['--setopt=install_weak_deps=True']
-config_opts['dnf_builddep_opts'] = config_opts.get('dnf_builddep_opts', []) + ['--setopt=install_weak_deps=True']
-config_opts['microdnf_builddep_opts'] = config_opts.get('microdnf_builddep_opts', []) + ['--setopt=install_weak_deps=True']
+config_opts[f'{config_opts.package_manager}_builddep_opts'] = config_opts.get(f'{config_opts.package_manager}_builddep_opts', []) + ['--setopt=install_weak_deps=True']
```

Comment:
Since this package also targets Python 2 it can't use f-strings.

Reply:
This is the mock configuration file, which is only processed in the Fedora/RHEL/CentOS container. It just happens to be Python because `mock` is written in Python, but it's really just a config file.

As distributed in EPEL 7+ and all current versions of Fedora, `mock` only supports Python 3.